Makefile

```diff
@@ -61,4 +61,4 @@ psr: src/vendor
 	cd src && composer dump-autoload -a
 
 i18n:
-	wp i18n make-pot src src/i18n/$(PLUGINSLUG).pot
+	wp i18n make-pot src src/framework/langs/$(PLUGINSLUG).pot
```

Comment:
Add them to i18n folder so that they are consistent over the projects, also fix the https://github.com/woocart/tax-handling/blob/master/src/better-tax-handling.php#L9 .

Reply:
@dz0ny changes done :)